fixtures/response/room_response_fixture.py

```diff
@@ -32,6 +32,23 @@
     }
 }
 
+get_room_response_query_by_date = '''
+query{
+    allRoomResponses(lowerDateLimit: "2019 feb 20",
+     upperDateLimit: "2019 Dec 25" ){
```

Comment:
You should take care of date validation in such a way that we can't have our upperDateLimit or loweDateLimit higher than the current date. This test should not pass in this case.

Reply:
Hello @RachaelNantale , you can use the function `validate current date` in tanalytics_helper.py to implement @michael-basweti 's feedback

Reply:
@nakatuddesuzan hope you meant `analytics_helper.py`

Reply:
Thanks @dnuwa, @michael-basweti, @nakatuddesuzan . This has been worked on.